dashboard/app/models/user_permission.rb

```diff
@@ -44,9 +44,8 @@ class UserPermission < ApplicationRecord
     PROGRAM_MANAGER = 'program_manager'.freeze,
     # Grants ability to be the instructor of any course no matter instructor_audience
     UNIVERSAL_INSTRUCTOR = 'universal_instructor'.freeze,
-    # Grants access to use AI Chat API
-    AI_CHAT_ACCESS = 'ai_chat_access'.freeze,
-
+    # Grants access to use AI Tutor which uses AI Chat API
+    AI_TUTOR_ACCESS = 'ai_tutor_access'.freeze,
```

Comment:
Do we envision projects other than ai-tutor needing access to this controller?

Reply:
TA will not need access to this controller because we are accessing OpenAI via Python. Not sure about other future uses, but I don't think this will be hard to change later when a next use case arises